app/pubspec.yaml

```diff
@@ -9,8 +9,11 @@ environment:
 dependencies:
   cached_network_image: ^3.4.1
   carousel_slider: ^5.0.0
+  file: ^7.0.1
   flutter:
     sdk: flutter
+  flutter_cache_manager: ^3.4.1
+  mocktail: ^1.0.4
```

Comment:
Are we using these 3 packages in this PR or is this just prep?

Reply:
Not anymore, it was used when I mocked the CacheManager for handle testing on FlexImage but now Im using another thing, good point